tests/api/entities/editions/create.test.js

```diff
@@ -36,6 +38,43 @@ describe('entities:editions:create', () => {
     .catch(done)
   })
 
+  it('should reject an edition without an ISBN without a publisher', async () => {
```

Comment:
if i understand this well, it restricts quite a lot edition creation, isnt it ? What if my old book's publisher is unknown to me ?

Reply:
@jum-s Are there books where the publisher is unknown? how else can we prevent the creation of  uneditable, unmergeable editions not connected to anything?

Reply:
a compromise could be to let it unrestricted on the server side but to request that the client pushes the user to avoid creating isolated editions, but that would mean that each client will have to re-implement this limitation, and that anyone directly hitting the API will be free to create isolated/junk editions

([example of an isolated edition that can't be edited or merged because we don't have an ISBN or the year of publication](https://inventaire.io/entity/inv:0888c0e93595f51c17f601eb6a85fed4))

Reply:
I removed the constrain in fa2cc519 to not block this PR on this